cmd/neofs-node/main.go

```diff
@@ -57,6 +58,12 @@ func main() {
 
 	c := initCfg(appCfg)
 
+	srv := initMetrics(c)
```

Comment:
but now we can just not have obsolete var and call `initMetrics` inside `preRunAndLog`'s args?

Reply:
done